packages/image_picker/android/build.gradle

```
@@ -48,7 +48,7 @@ android {
         disable 'InvalidPackage'
     }
     dependencies {
-        api 'androidx.legacy:legacy-support-v4:1.0.0'
+        api 'androidx.core:core:1.0.2'
```

Good catch, thanks!

Do you mind also changing this to an `implementation` instead of an `api` dependency? That way we also wouldn't be exposing the dep here like we are now.

```suggestion
        implementation 'androidx.core:core:1.0.2'
```